specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -0,0 +1,347 @@
+      "scopes": {
+        "https://atlas.microsoft.com/.default": "https://atlas.microsoft.com/.default"
+      }
+    },
+    "AzureKey": {
+      "type": "apiKey",
+      "description": "This is a shared key that is provisioned when creating an [Azure Maps resource](https://aka.ms/amauth) through the Azure management plane  via Azure portal, PowerShell, CLI, Azure SDKs, or REST APIs.\n\n With this key, any application is authorized to access  all REST APIs. In other words, these can currently be treated as master keys to the account which they are issued for.\n\n For publicly exposed applications, our recommendation is to use server-to-server access of Azure Maps REST APIs where this key can be  securely stored.",
+      "name": "subscription-key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://atlas.microsoft.com/.default"
+      ]
+    },
+    {
+      "AzureKey": []
+    }
+  ],
+  "responses": {},
+  "parameters": {
+    "RenderApiVersionV2": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API. Current version is 2.1",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2.1",
+      "x-ms-parameter-location": "client"
+    }
+  },
+  "paths": {
+    "/map/tile": {
+      "get": {
+        "description": "**Applies to**: S0 and S1 pricing tiers.\n\nThe Get Map Tiles API allows users to request map tiles in vector or raster formats typically to be integrated  into a map control or SDK. Some example tiles that can be requested are Azure Maps road tiles, real-time  Weather Radar tiles or the map tiles created using [Azure Maps Creator](https://aka.ms/amcreator). By default,  Azure Maps uses vector tiles for its web map control (Web SDK) and Android SDK.",
+        "operationId": "RenderV2_GetMapTilePreview",
+        "x-ms-examples": {
+          "GetMapTile": {
+            "$ref": "./examples/GetMapTileV2.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "../2.0/render.json#/parameters/RenderApiVersionV2"
```

Comment:
We have `RenderApiVersionV2` in this swagger with default value: `api-version=2.1`. Use this one instead?

Reply:
oh that's a pretty bad typo, thanks